perception/autoware_shape_estimation/lib/corrector/reference_shape_size_corrector.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Autoware Foundation. All rights reserved.
```

Comment:
```suggestion
// Copyright 2024 Tier IV, Inc.. All rights reserved.
```

Reply:
Thank you very much. Comment has been fixed.